awapi/autoware_api_msgs/CMakeLists.txt

```
@@ -1,39 +1,29 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(autoware_api_msgs)
 
-add_compile_options(-std=c++14)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  std_msgs
-  geometry_msgs
-  diagnostic_msgs
-  autoware_planning_msgs
-)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
 
-add_message_files(
-  DIRECTORY msg
-  FILES
-    Latlon.msg
-    Euler.msg
-    AwapiVehicleStatus.msg
-    AwapiAutowareStatus.msg
-    LaneChangeStatus.msg
-    ObstacleAvoidanceStatus.msg
-)
+find_package(ament_cmake REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(diagnostic_msgs REQUIRED)
+find_package(autoware_planning_msgs REQUIRED)
```

We can replace this with 

```suggestion
find_package(ament_cmake_auto REQUIRED)
ament_auto_find_build_dependencies()
```

---

Sure, done.